app/views/layouts/application.html.erb

```diff
@@ -7,10 +7,30 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_importmap_tags %>
   </head>
 
   <body>
-    <%= yield %>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
+    <div class="container">
+      <div class="nav menu">
+        <div class="menu">
+          <%= image_tag('menu-icon.png', :alt => 'menu', :width => '35px', :height => '35px') %>
+        </div>
+        <div class="nav-text">
+          <h3>Budget<span>App</span></h3>
+        </div>
```

Comment:
- [ ] OPTIONAL: It would be awesome to have this logo as a link that takes users to main page

Reply:
Hi @ShoiraTa 

Thank you, I have fixed that. Thanks